profiles-sdk-examples/build-profiles/README.md

```diff
@@ -0,0 +1,38 @@
+# Build Profiles
+
+This example is a CLI application for building Cortex Profiles. This builds off
+the [Local Clients](../local-clients/README.md) example for its setup 
+(see [ProfileSchemas](../local-clients/README.md#profile-schemas)).
```

Comment:
Points to a description of the ProfileSchemas used in the example.. Ended up deleting this because I had the same link at the start of the next section.